tests/test_pytorch_transformer_parity.py

```diff
@@ -172,7 +172,7 @@ def test_pytorch_tranformer_parity(device=torch.device("cuda")):
             dim_feedforward=4 * EMB,
             dropout=DROP,
             activation=ACTIVATION,
-            layer_norm_eps=1e-05,
```

Comment:
should have been part of https://github.com/facebookresearch/xformers/pull/221 (1e-6 is the default everywhere now), unrelated to this PR but I spotted that while writing this one